Source/SwiftLintFramework/Models/Command.swift

```diff
@@ -84,15 +84,16 @@ public struct Command: Equatable {
                 return nil
         }
         self.action = action
-        ruleIdentifiers = scanner.string.bridge()
-            .substring(from: scanner.scanLocation + 1)
-            .components(separatedBy: .whitespaces)
         line = lineAndCharacter.line
         character = lineAndCharacter.character
 
-        let hasModifier = actionAndModifierScanner.scanString(string: ":") != nil
+        let ruleTexts = scanner.string.bridge().substring(
+            from: scanner.scanLocation + 1
+        ).components(separatedBy: .whitespaces)
+        ruleIdentifiers = Set(ruleTexts.map { RuleIdentifier($0) })
```

Comment:
`.map(RuleIdentifier.init)`?

Reply:
This would be ambiguous since there are two inits with string available...

Reply:
You can use the qualified symbol name. For instance, `RuleIdentifier.init(_:)` to specify the one without label. This way you can use map - e.g. `.map(RuleIdentifier.init(_:))`.

Reply:
Thanks, implemented that one